pkgs/development/tools/minipro/unstable.nix

```
@@ -0,0 +1,49 @@
+{ stdenv, installShellFiles, which, fetchgit, git, pkg-config, libusb1 }:
+stdenv.mkDerivation rec {
+  pname = "minipro";
+  version = "0.4dev";
+
+  src = fetchgit {
+    url = "https://gitlab.com/DavidGriffith/minipro.git";
+    rev = "896fe7d9d6f1e77931626d273e880856884ee6d8";
+    sha256 = "1b14kg6aj23a808lngap63qf92193rx7s7l94ymf9dkwqa1xb524";
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    installShellFiles
+  ];
+
+  buildInputs = [
+    stdenv
+    which
+    libusb1
+    git
+  ];
+
+  preConfigure = ''
+    export PKG_CONFIG="${pkg-config}/bin/pkg-config";
+    substituteInPlace Makefile --replace '$(shell date "+%Y-%m-%d %H:%M:%S %z")' "1970-01-01 00:00:00 +0000"
+  '';
+
+  buildPhase = ''
+    make -e minipro
+  '';
+
+  installPhase = ''
+    mkdir $out/bin $out/lib/udev/rules.d -pv
+    cp -rv ./minipro $out/bin/
+    mv udev/* $out/lib/udev/rules.d/ -v
+    installManPage ./man/minipro.1
+    installShellCompletion --bash ./bash_completion.d/minipro
+  '';
+
+  meta = with stdenv.lib; {
+    description = "An open source program for controlling the MiniPRO TL866xx series of chip programmers";
+    homepage = "https://gitlab.com/DavidGriffith/minipro";
+    maintainers = with maintainers; [ kalium ];
+    license = stdenv.lib.licenses.gpl3;
```

Since you're already doing `with stdenv.lib;`, you don't need to mention that here.
```suggestion 
    license = licenses.gpl3;
```